pretyping/inductiveops.mli

```diff
@@ -192,6 +192,10 @@ val find_rectype     : env -> evar_map -> EConstr.types -> inductive_type
 val find_inductive   : env -> evar_map -> EConstr.types -> (inductive * EConstr.EInstance.t) * constr list
 val find_coinductive : env -> evar_map -> EConstr.types -> (inductive * EConstr.EInstance.t) * constr list
 
+(** [instantiate_constructor_params ind mind i params] instantiates the
+    type of the i-th constructor (counting from 1) with parameters [params] *)
+val instantiate_constructor_params : pinductive -> Inductive.mind_specif -> int -> constr list -> constr
```

Comment:
Nitpick: I think it would be better to take a pconstructor rather than a pinductive and an integer, because this signal semantic intent in the index that could be enforced by typing one day, whereas now we have to rely on the comment.

Reply:
I agree and made the same thing by transitivity to `mis_nf_constructor_type` (exported w/o being used though).